test/fastboot-test.js

```diff
@@ -409,19 +409,25 @@ describe('FastBoot', function() {
     });
 
     let result = await fastboot.visit('/', { buildSandboxPerVisit: true });
+    let analytics = result.getAnalytics();
     let html = await result.html();
 
     expect(html).to.match(/Items: 1/);
+    expect(analytics.sandboxPerVisit.isDegradedRequest).to.be.false;
```

Comment:
Can we change these tests to deep equals (asserting the full contents of `analytics`)?